core/modules/layout/includes/block.text.inc

```diff
@@ -76,10 +76,10 @@ class BlockText extends Block {
       $form['convert']['reusable'] = array(
         '#type' => 'checkbox',
         '#title' => t('Make this block reusable'),
-        '#description' => t('If enabled, this block will be made reusable across different layouts and be listed in on the !block_link page.', array('!block_link' => l(t('Custom blocks'), 'admin/structure/block'))),
+        '#description' => t('If enabled, this block will be made reusable across different layouts and will be listed in the !link page. This option may not be unchecked once enabled.', array('!link' => l(t('Custom blocks'), 'admin/structure/block'))),
         '#weight' => 1,
       );
-      $form['convert']['label'] = array(
+      $form['convert']['reusable-admin-label'] = array(
```

Comment:
Hyphens are uncommon in form element "name" keys like this. Underscores should be used. But really, do we need to change these keys at all? Seems like we would minimize impact by leaving `reusable-admin-` off these keys, and it doesn't seem to affect the remainder of the changes.

Reply:
agreed